scripts/pre_push_hook.py

```diff
@@ -330,6 +331,27 @@ def main():
             if frontend_status != 0:
                 print 'Push aborted due to failing frontend tests.'
                 sys.exit(1)
+    try:
+        with open('successful_push_counter.txt', 'r+') as f:
+            successful_push_count = f.read()
+    except IOError:
+        with open('successful_push_counter.txt', 'w') as f:
+            successful_push_count = '0'
+            f.write(successful_push_count)
+    if int(successful_push_count) < SUCCESSFUL_PUSH_THRESHOLD:
+        sys.stdin.flush()
+        sys.stdin = open('/dev/tty', 'r')
+        print ('Please confirm that you have performed all the tests by'
```

Comment:
I think you haven't understood the issue yet. It is not about running `scripts/run_tests.sh`, we want the developer to test his/her changes _manually_ before pushing. (This is clearly mentioned in the corresponding issue).